upcoming-release-notes/2909.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [lelemm]
+---
+
+Added schedules to API
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏some folks read the release notes, so best if we can be a bit more verbose here.

```suggestion
Added schedules methods to the API package.
```